operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -532,8 +532,8 @@ function diff_model() {
     org.python.util.jython \
     ${SCRIPTPATH}/model_diff.py $1 $2 > ${WDT_OUTPUT} 2>&1
   if [ $? -ne 0 ] ; then
-    trace SEVERE "Failed to compare models. Check logs for error."
-    trace SEVERE "$(cat ${WDT_OUTPUT})"
+    trace SEVERE "Failed to compare models. Check logs for error. Comparison output:"
+    cat ${WDT_OUTPUT}
```

Comment:
Implementation note:  When the introspector java parses out a SEVERE it includes everything after the SEVERE up to the next trace statement.  (So the 'cat' above is included in the SEVERE just before it.)